addon/components/time-series-chart.js

```diff
@@ -61,6 +61,15 @@ const TimeSeriesChartComponent = ChartComponent.extend(LegendMixin,
   // Force X-Axis labels to print vertically
   xAxisVertLabels: false,
 
+  willDestroyElement: function() {
+    var groups = this.get('groups');
```

Comment:
why in some cases you run `groups.on('mouseover', null);` and in some cases we don't?

Reply:
I never used d3, so maybe I do something wrong. I'm removing the event handler on the same selection that it was applied to, which varies from component to component.